index2.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style2.css">
+
+    <title>Document</title>
+</head>
+<body>
+   <div class="header">
+    <img class="cell-info" src="Light.png"></div>
+    <div class="bottom-section">
+    <p class="welcome">Welcome</p>
+    <img class="union" src="Union.png">
+    </div>
+   </div>
+
+   <div class="flex">
+   <img src="background.png">
+    <div class="greeting">
+        <p class="hi">Hey there. I'm</p>
+        <p class="first-name">Mariane</p>
+        <p class="job">I am a software</p>
+        <p class="job-info">Developer</p>
+     </div>
+     <div class="information-box">
+        <p class="info">I can help you build a product,</p>
+        <p class="info2">feature or website Look through</p>
+        <p class="info3"> some of my work and experience! </p>
+        <p class="info4"> If you like what you see and have</p>
+        <p class="info5">a project you need coded</p>
+        <p class="info6"> don’t hestiate to contact me.</p>
+      </div>
+    </div>
```

Comment:
- [ ] Kindly change this div wrapper to something more semantical like `<section>` to make your HTML more professional.